src/_playground/Routes.js

```diff
@@ -345,7 +361,8 @@ export class Routes extends Component {
                             {navItems}
                         </nav>
                     </div>
-                    <div className='frDocs-Content'>
+                    <div className={`frDocs-Content ${showNav}`}>
```

Comment:
A cleaner way of implementing conditional CSS classes is to use the `classnames` package.  You can check a number of other components for usage examples.